basic_module/account/src/types.rs

```diff
@@ -0,0 +1,95 @@
+use ccrypto::blake256;
+use ckey::{Ed25519Public as Public, NetworkId, Signature};
+use primitives::H256;
+
+#[allow(dead_code)]
+pub type ErroneousTransactions = Vec<SignedTransaction>;
+
+#[derive(Clone)]
+pub struct Account {
+    pub balance: u64,
+    pub sequence: u64,
+}
+
+// Temporary measures for build.
+// This function will be removed when the structure of the context and conversion logic is completed.
+impl From<Vec<u8>> for Account {
+    fn from(_vec: Vec<u8>) -> Account {
```

Comment:
Let's complete the implementation of the `serialization` and `deserialization`. FML provides the handle type `ser` and `de ` and resultant type is Bytes for the `SubStorageAccess`. Now, you need to implement the module level `ser` and `de` as you want.